project/Build.scala

```diff
@@ -738,6 +740,31 @@ object Build {
         (unmanagedSourceDirectories in (`dotty-library-bootstrapped`, Compile)).value,
     )
 
+  lazy val `dotty-staging` = project.in(file("staging")).
+    withCommonSettings(Bootstrapped).
+    dependsOn(dottyCompiler(Bootstrapped)).
+    dependsOn(dottyCompiler(Bootstrapped) % "test->test").
+    settings(commonBootstrappedSettings).
+    settings(
+      fork in run := true,
+      fork in Test := true,
+      javaOptions ++= {
+        val attList = (dependencyClasspath in Runtime).value
+        val jars = packageAll.in(`dotty-compiler-bootstrapped`).value
+        List(
```

Comment:
Too much duplication with the compiler javaOptions, either use the same javaOptions in both projects, or make a `def` to output the common parts.

Reply:
I extracted in a common setting